modules/ROOT/pages/plugins/opensource/image.adoc

```diff
@@ -60,17 +60,17 @@ include::partial$configuration/image_title.adoc[]
 
 include::partial$configuration/image_uploadtab.adoc[]
 
-= include::partial$configuration/images_file_types.adoc[]
+include::partial$configuration/images_file_types.adoc[leveloffset=+1]
```

Comment:
```suggestion
include::partial$configuration/images_file_types.adoc[]
```

As I mentioned via DM this here actually shouldn't have this. It was also incorrect in the old system and is wrong on production today as well.